pkg/util/vmopv1/vmgroup_test.go

```diff
@@ -355,3 +356,180 @@ var _ = Describe("UpdateGroupLinkedCondition",
 		})
 	},
 )
+
+var _ = Describe("RemoveStaleGroupOwnerRef", func() {
+	const (
+		apiVersion = "vmoperator.vmware.com/v1alpha5"
```

Comment:
nit: could be replaced with `vmopv1.GroupVersion.String()` so no need to update this when we bump to newer version